.travis.yml

```diff
@@ -1,23 +1,25 @@
 language: python
-python: 2.7
-sudo: false
+sudo: true
 cache:
+  pip: true
   directories:
-  - buildout-cache
+    - eggs
+    - downloads
+python:
+  - 2.7.13
 env:
-  - PLONE_VERSION=4.2
   - PLONE_VERSION=4.3
   - PLONE_VERSION=5.0
-matrix:
-  fast_finish: true
+  - PLONE_VERSION=5.1
 install:
-  - sed -ie "s#travis-4.x.cfg#travis-$PLONE_VERSION.x.cfg#" travis.cfg
-  - if test "$PLONE_VERSION" = "4.2"; then echo >> travis.cfg; echo "Products.membrane = 2.1.13" >> travis.cfg; fi
-  - mkdir -p buildout-cache/downloads
-  - pip install zc.buildout
-  - buildout -c travis.cfg annotate
-  - buildout -c travis.cfg -N -q -t 3
-before_script:
-  - export DISPLAY=:99.0
-  - sh -e /etc/init.d/xvfb start
-script: bin/test
+  - virtualenv .
+  - bin/pip install -r requirements.txt
+  - sed -ie "s#plone-5.1.x.cfg#plone-$PLONE_VERSION.x.cfg#" travis.cfg
+  - bin/buildout -c travis.cfg buildout:download-cache=downloads code-analysis:return-status-codes=True
```

Comment:
please add a `bin/buildout annotate` above he so we can check that the right versions are being tested.